### DISCUSSION TITLE
[Question] Is ARM64 a supported architecture?

### DISCUSSION THREAD

**Discussion by emiliofernandes** [Electrostatics/apbs#254] [General]
Hello,

I’d like to ask whether APBS is supported on Linux/Mac ARM64 systems ?
I need to use https://github.com/RashmiKumari/g_mmpbsa which depends on APBS.

Thank you!

Emilio

**Comment by sobolevnrm** [comment]
APBS works on most flavors of Linux.  We haven't specifically compiled a binary for the ARM64, however.

**Reply by emiliofernandes** [reply]
Thank you for the answer, @sobolevnrm !
I will try to build it and let you know how it goes!

**Reply by emiliofernandes** [reply]
I need some help here!
I try to build APBS on Ubuntu 20.04.5 ARM64 and I follow the steps at https://github.com/Electrostatics/apbs/blob/main/.github/workflows/build.yaml


`./.build.sh` fails with:
```
-- Could NOT find BLAS (missing: BLAS_LIBRARIES) 
CMake Error at build/_deps/fetk-src/cmake/FindSuiteSparse.cmake:139 (message):
  Failed to find SuiteSparse - Did not find BLAS library (required for
  SuiteSparse).
Call Stack (most recent call first):
  build/_deps/fetk-src/cmake/FindSuiteSparse.cmake:286 (suitesparse_report_not_found)
  build/_deps/fetk-src/mc/CMakeLists.txt:38 (find_package)
```

https://github.com/Electrostatics/apbs/blob/e8d1a9ce9a4e4f2d9f0a186eb1ae09bdeabbe72e/.github/workflows/build.yaml#L145 installs just ` libopenblas-serial-dev`. What else needs to be installed ?

I tried with `libopenblas-dev` and `libblas-dev` but neither helped.

**Reply by emiliofernandes** [reply]
Could this be related ?
```
Process XSDK defaults ...
USE_XSDK_DEFAULTS = 'FALSE'
-- SuperLU will be built as a static library.
-- Did not find or specify BLAS so configure to build internal CBLAS ...
CMake Error at build/_deps/superlu-src/CBLAS/CMakeLists.txt:82 (add_library):
  add_library cannot create target "blas" because another target with the
  same name already exists.  The existing target is a static library created
  in source directory
  "/home/ubuntu/git/apbs/build/_deps/fetk-src/punc/src/blas".  See
  documentation for policy CMP0002 for more details.
```
